Disable dbus dependency. Autodetecting isn't deterministic, we don't have the DEPENDS
and we don't want to built this piece so hardcode it off.

Upstream-Status: Inappropriate [configuration]

RP 2012/2/25

Index: glib-2.30.2/configure.ac
===================================================================
--- glib-2.30.2.orig/configure.ac	2012-02-25 14:50:07.444242397 +0000
+++ glib-2.30.2/configure.ac	2012-02-25 14:51:28.096240485 +0000
@@ -3769,10 +3769,9 @@
 # Check for libdbus1 - Optional - is only used in the GDBus test cases
 #
 # 1.2.14 required for dbus_message_set_serial
-PKG_CHECK_MODULES(DBUS1,
-                  dbus-1 >= 1.2.14,
-                  [AC_DEFINE(HAVE_DBUS1, 1, [Define if dbus-1 is available]) have_dbus1=yes],
-                  have_dbus1=no)
+DBUS1_CFLAGS=
+DBUS1_LIBS=
+$have_dbus1=no      
 AC_SUBST(DBUS1_CFLAGS)
 AC_SUBST(DBUS1_LIBS)
 AM_CONDITIONAL(HAVE_DBUS1, [test "x$have_dbus1" = "xyes"])
